source/walkthroughs/deploy/deploy_updates.html.erb

```diff
@@ -33,11 +33,11 @@ substep = 0
       In order to provide you appropriate instructions, please choose your Meteor version: <%= render_partial("/shared/meteor_picker.html") %>
     </p>
     <p>
-      Inside your application's code directory, on your local computer, use the <code class="meteor old">meteor bundle</code><code class="meteor new">meteor build</code> command to create a Meteor package tarball of the latest application code.
+      Inside your application's code directory, on your local computer, use the <code class="meteor v13">meteor bundle</code><code class="meteor v14 v16">meteor build</code> command to create a Meteor package tarball of the latest application code.
```

Comment:
The choice between 1.4 and 1.6 makes no difference in this page, which means that users of 1.4 will have to perform a useless click. The picker here should be <= 1.3.5.1 and >= 1.4.